src/main/java/br/com/caelum/vraptor/util/jpa/extra/ParameterLoader.java

```diff
@@ -69,32 +73,29 @@ public ParameterLoader(EntityManager em, HttpServletRequest request, ParameterNa
 		this.flash = flash;
 	}
 
+	@Accepts
 	public boolean containsLoadAnnotation(ControllerMethod method) {
```

Comment:
you can ask `ControllerMethod` injected on constructor, instead of both `accepts` 
and `aroundcall` methods (it also improve performance, avoiding manual injection)